docker/ubi/docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (C) 2022 The Falco Authors.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -x
+rm -fr /usr/src/kernels/ && rm -fr /usr/src/debug/
+rm -fr /lib/modules && ln -s $HOST_ROOT/lib/modules /lib/modules
+rm -fr /boot && ln -s $HOST_ROOT/boot /boot
```

Comment:
Should those lines be moved into the below _if_ branch?

Moreover, could you explain why those symlinks are required? I'd expect that `falco-driver-loader` would correctly handle them? Am I missing something?
If there's a valid reason, perhaps it is worth adding a comment in the code, so we won't forget it :smile_cat:

Reply:
Without the symlinks, dkms can't find the requisite code (I think headers) to build.... for example, with those lines commented out we get the following error:

```bash ./runFalcoLive 
* Setting up /usr/src links from host
* Running falco-driver-loader for: falco version=0.31.1, driver version=b7eb0dd65226a8dc254d228c8d950d07bf3521d2
* Running falco-driver-loader with: driver=module, compile=yes, download=yes
* Unloading falco module, if present
* Looking for a falco module locally (kernel 5.4.0-94-generic)
* Trying to download a prebuilt falco module from https://download.falco.org/driver/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/falco_ubuntu-generic_5.4.0-94-generic_106.ko
curl: (22) The requested URL returned error: 404 
Unable to find a prebuilt falco module
* Trying to dkms install falco module with GCC /usr/bin/gcc
DIRECTIVE: MAKE="'/tmp/falco-dkms-make'"
Creating symlink /var/lib/dkms/falco/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/source -> /usr/src/falco-b7eb0dd65226a8dc254d228c8d950d07bf3521d2
* Running dkms build failed, couldn't find /var/lib/dkms/falco/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/build/make.log (with GCC /usr/bin/gcc)
* Trying to load a system falco module, if present
Consider compiling your own falco driver and loading it or getting in touch with the Falco community
Fri Mar 18 15:38:39 2022: Falco version 0.31.1 (driver version b7eb0dd65226a8dc254d228c8d950d07bf3521d2)
Fri Mar 18 15:38:39 2022: Falco initialized with configuration file /etc/falco/falco.yaml
Fri Mar 18 15:38:39 2022: Loading rules from file /etc/falco/falco_rules.yaml:
Fri Mar 18 15:38:39 2022: Loading rules from file /etc/falco/falco_rules.local.yaml:
Fri Mar 18 15:38:40 2022: Loading rules from file /etc/falco/k8s_audit_rules.yaml:
Fri Mar 18 15:38:40 2022: Unable to load the driver.
Fri Mar 18 15:38:40 2022: Runtime error: error opening device /host/dev/falco0. Make sure you have root credentials and that the falco module is loaded.. Exiting.
```

Reply:
This is something we had to add to the sysflow collector to get it to work.   Seems to be an issue related to redhat directory structure.